03.api.md

```diff
@@ -358,92 +358,973 @@ jerry_api_gc ();
 ```
 
 
+# jerry_api_eval
+
+**Summary**
+Perform JavaScript `eval`.
+
+**Prototype**
+
+```c
+jerry_completion_code_t
+jerry_api_eval (const jerry_api_char_t *source_p,
+                size_t source_size,
+                bool is_direct,
+                bool is_strict,
+                jerry_api_value_t *retval_p);
+```
+
+- `source_p` - source code to evaluate;
+- `source_size` - length of the source code;
+- `is_direct` - whether to perform `eval` in "direct" mode (possible only if `eval` invocation is performed from native function, called from JavaScript);
```

Comment:
No, this is incorrect. Direct mode means it is called as "eval" and not through some alias. In this case the context is the same as the caller context. I would not expose this flag, because it has no meaning when called from C. Actually it may messes an unexpected context.